### ISSUE TITLE
1.4.8 Visual Presentation += Fonts

### ISSUE THREAD

**Issue by LaZay** [w3c/wcag3#161]
As the 1.4.8 Visual Presentation criteria is at AAA level, can we add font selection into user customization? This is particularly useful for dyslexic people, but not only. It is also usefull for reading comfort.  As a matter of fact, most publishers still use paper fonts with serif in their digital products, though it is not recommanded (blurring effect on screen).

Font families listed in user selection should be available on the user's computer of course (ex: [OpenDyslexic](https://en.wikipedia.org/wiki/OpenDyslexic)). This is an additional difficulty for implementation, compared to other CSS properties. But this overloading of author's CSS font families would be great.

**Comment by LaurenceZaysser** [comment]
I agree with you, but so far, WCAG does not make a clear distinction between criteria depending on content (Web format + accessible content) and on reading systems (ex:  [https://www.w3.org/TR/WCAG22/#resize-text](1.4.4 Resize Textamong others). Maybe this is another request to the WCAG?

**Comment by adampage** [comment]
Hi @LaurenceZaysser, thank you for raising this issue. The working group is currently developing a “[Text style adjustable](https://w3c.github.io/wcag3/guidelines/#text-style-adjustable)” requirement which includes font face customization.
